src/langs/ar.json

```diff
@@ -1,17 +1,36 @@
 {
-  "GRAASP_PLAYER": "مشغل چراسپ ",
+  "GRAASP_PLAYER": "مشغل graasp ",
   "SHOW_CHAT_TOOLTIP": "إظهار الدردشة",
   "HIDE_CHAT_TOOLTIP": "إخفاء الدردشة",
+  "EXIT_FULLSCREEN_TOOLTIP": "الخروج من ملء الشاشة ",
+  "ENTER_FULLSCREEN_TOOLTIP": "أدخل ملء الشاشة",
   "SHOW_PINNED_ITEMS_TOOLTIP": "إظهار العناصر المثبتة",
   "HIDE_PINNED_ITEMS_TOOLTIP": "إخفاء العناصر المثبتة",
   "PINNED_ITEMS": "العناصر المثبتة",
   "LOAD_MORE": "تحميل المزيد",
   "HIDDEN_WRAPPER_TOOLTIP": "هذا العنصر مخفي، يمكنك رؤيته لأن لديك حق الوصول للمسؤول أو للكتابة، ولن يتمكن المستخدمون الذين لديهم حق الوصول للقراءة من رؤيته",
   "SHOW_MORE": "أظهر المزيد...",
   "RECENT_ITEMS_TITLE": "أحدث العناصر",
-  "DRAWER_ARIAL_LABEL": "درج مفتوح",
+  "DRAWER_ARIAL_LABEL": "افتح الواجهة",
   "ERROR_FETCHING_ITEM": "حدث خطأ أثناء جلب العنصر المطلوب",
   "ERROR_ACCESSING_ITEM": "لا يمكنك الوصول إلى هذا العنصر",
   "ERROR_ACCESSING_ITEM_HELPER": "حسابك الحالي ليس لديه حقوق الوصول إلى هذا العنصر.",
-  "SIGN_IN_BUTTON_TEXT": "تسجيل الدخول"
+  "SIGN_IN_BUTTON_TEXT": "تسجيل الدخول",
+  "FALLBACK_TITLE": "عفواً",
+  "FALLBACK_TEXT": "هناك خطأ ما. حاول مرة اخرى. إذا استمرت المشكلة اتصل بنا.",
+  "FALLBACK_RELOAD_PAGE": "إعادة تحميل الصفحة",
+  "ITEM_ID_NOT_VALID": "معرف العنصر غير صالح، يرجى التحقق منه مرة أخرى. انقر على الزر أدناه للعودة إلى الصفحة الرئيسية.",
+  "GO_TO_HOME": "اذهب إلى الصفحة الرئيسية",
+  "ITEM_CHATBOX_TITLE": "\"{{name}}\" دردشة",
+  "ITEM_TITLE_CREATED_AT": "تاريخ الإنشاء: {{date}}",
+  "ITEM_TITLE_UPDATED_AT": "تاريخ التحديث: {{date}}",
+  "HOME_PAGE_TITLE": "أحدث العناصر",
+  "NAVIGATION_ISLAND_CHAT_BUTTON_HELPER_TEXT_READERS": "لم يتم تفعيل مربع الدردشة لهذا العنصر.",
+  "NAVIGATION_ISLAND_CHAT_BUTTON_HELPER_TEXT_WRITERS": "لتفعيل مربع الدردشة، انتقل إلى إعدادات العنصر في Builder وقم بتنشيط إعداد \"إظهار الدردشة في المشغل\".",
+  "NAVIGATION_ISLAND_PINNED_BUTTON_HELPER_TEXT_READERS": "لا توجد عناصر مثبتة لهذه الصفحة.",
+  "NAVIGATION_ISLAND_PINNED_BUTTON_HELPER_TEXT_WRITERS": "لإضافة عناصر إلى القسم المثبت، قم بتعيينها كمثبتة في واجهة Builder.",
+  "TREE_NAVIGATION_RELOAD_TEXT": "أعد التحميل !",
```

Comment:
```suggestion
  "TREE_NAVIGATION_RELOAD_TEXT": "! أعد التحميل",
```

Reply:
maybe it doesn't look good, but It's already from right to left, so once it's displayed it's going to fit properly